scripts/vs/buildimage-vs-image/runtest.sh

```diff
@@ -40,67 +22,61 @@ export ANSIBLE_LIBRARY=/data/sonic-mgmt/ansible/library/
 # workaround for issue https://github.com/Azure/sonic-mgmt/issues/1659
 export export ANSIBLE_KEEP_REMOTE_FILES=1
 
-PYTEST_COMMON_OPTS="--inventory veos.vtb \
-                    --host-pattern all \
-                    --user admin \
-                    -vvv \
-                    --show-capture stdout \
-                    --testbed $tbname \
-                    --testbed_file vtestbed.csv \
-                    --disable_loganalyzer \
-                    --log-file-level debug"
+PYTEST_CLI_COMMON_OPTS="\
+    -i veos.vtb \
+    -d $dut \
+    -n $tbname \
+    -f vtestbed.csv \
+    -k debug \
+    -l warning \
+    -m group \
+    -e --disable_loganalyzer
+"
 
-# Check testbed health
 cd /data/sonic-mgmt/tests
-rm -rf logs results
+rm -rf logs
 mkdir -p logs
-mkdir -p results
-py.test $PYTEST_COMMON_OPTS --log-file logs/test_nbr_health.log --junitxml=results/tr.xml test_nbr_health.py
```

Comment:
do we check nbr health now if this is removed?

Reply:
Yep, also part of the pretest steps! @yxieca can confirm the exact behavior.